### ISSUE TITLE
[ui] FR: option to hide a search tile if there are no items to display

### ISSUE THREAD

**Issue by madduck** [eikek/docspell#2275]
Subject says it all. If there are no documents tagged `urgent`, then there is no need to display the tile that shows urgent documents. A checkbox for each tile would do.

**Comment by eikek** [comment]
A bit unsure what part you mean (what is "tile"): are you talking about dashboards?

**Comment by madduck** [comment]
Yes, a tile is a Kachel is one of those thingies that make up dashboards. I am saying that if there are no "new" documents, why does the "new" tile have to be shown. This might make sense in some cases, but in others, not having the tile appear at all certainly would help conserve screen estate.

![image](https://github.com/eikek/docspell/assets/195073/ceafb745-c9d0-4748-a790-2947e860ea88)

Okay, I get it. The smiley had me smile. I never saw that before, actually ;)

**Comment by eikek** [comment]
Yes the smiley is when checking your inbox and there is nothing in it - no new invoices 😆 

I see - as you said it might be good and bad depending on who you ask. People might be confused that a tile goes away suddenly or so. But yeah, we can add an option to hide it when empty.
